package.json

```diff
@@ -17,7 +17,7 @@
     "prebid"
   ],
   "globalVarName": "pbjs",
-  "analytics": [],
+  "analytics": ["sharethrough_analytics"],
```

Comment:
Analytics is explicitly turned on by the publisher so this array should be blank by default